dockerd-patches/buildkit-apparmor.patch

```diff
@@ -0,0 +1,291 @@
+From 5472f39022e99c14b2f055eac4d9619e3663ae20 Mon Sep 17 00:00:00 2001
```

Comment:
Including this patch file is hopefully temporary pending #13 -- I'd much rather update to 19.03.15 than carry this patch manually but this is what's needed to make this testing actually possible. :sweat_smile: